web/src/main/java/org/springframework/security/web/FilterChainProxy.java

```diff
@@ -147,6 +154,12 @@ public class FilterChainProxy extends GenericFilterBean {
 
 	private static final String FILTER_APPLIED = FilterChainProxy.class.getName().concat(".APPLIED");
 
+	static final String REQUEST_OBSERVATION_NAME = "http.requests";
+
+	static final String UNSECURED_OBSERVATION_NAME = "http.unsecured.requests";
```

Comment:
Can we match these to what MVC uses, like  `http.server.requests.secured` and `http.server.requests.unsecured`?

Reply:
I'd prefer to have a simple way to turn all Spring Security observations off. Not that it can't be done another way, but right now I've just updated the PR so that all start with `spring.security.`. In this way, an application can create a very simple predicate to turn on Spring Security observations.

I see the value of aligning, too... it would be nice if there were a way to get both. One way may be to introduce a class like `SpringSecurityObservationPredicates#noObservations()` or similar. Is there another potential route?

Reply:
I like the `SpringSecurityObservationPredicates#noObservations()` approach.